.github/workflows/build-release.yml

```
@@ -0,0 +1,68 @@
+name: Build and Release
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    # Check all PR
+
+jobs:
+  build:
+    name: Build distribution
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: setup Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: "3.12"
+    - run: python -m pip install tox
+    - name: Build package
+      run: tox -e build
+      env:
+        # Use the CPU only version of torch when building/running the code
+        PIP_EXTRA_INDEX_URL: https://download.pytorch.org/whl/cpu
+    - name: Store the distribution packages
+      uses: actions/upload-artifact@v4
+      with:
+        name: python-package-distributions
+        path: dist/
+
+  publish-to-pypi:
+    name: Publish to PyPI
+    if: startsWith(github.ref, 'refs/tags/v')
+    needs: build
+    runs-on: ubuntu-latest
+    environment:
+      name: pypi
+      url: https://pypi.org/project/torch-pme
+    permissions:
+      id-token: write
+
+    steps:
+      - name: Download all the dists
+        uses: actions/download-artifact@v4
+        with:
+          name: python-package-distributions
+          path: dist/
+      - name: Publish distribution to PyPI
+        uses: pypa/gh-action-pypi-publish@release/v1
+
+  github-release:
+    name: GitHub Release
+    needs: publish-to-pypi
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download all the dists
+        uses: actions/download-artifact@v4
+        with:
+          name: python-package-distributions
+          path: dist/
+      - name: Publish to GitHub release
+        uses: softprops/action-gh-release@v2
+        with:
+          files: >-
+            dist/*.tar.gz
+            dist/*.whl
+            prerelease: ${{ contains(github.ref, '-rc') }}
```

```suggestion
        with:
          files: >-
            dist/*.tar.gz
            dist/*.whl
          prerelease: ${{ contains(github.ref, '-rc') }}
```

Not sure what ` >-` is doing

---

I thunk it is the same as `|`

---

My understanding is `|` creates multiple lines, keeping the new lines, and `>` strips the new lines so 

```
key: >
   aa
   bb 
   cc
```

Is the same as 

```
key: aa bb cc
```

`>-` I don't know. Could it create a list from multiple lines? I though this field should be a list.

---

Hmm could be it seems to generally work as in the PYPI example 

https://packaging.python.org/en/latest/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/#the-whole-ci-cd-workflow